src/component/utils/KeyBindingUtil.js

```diff
@@ -15,6 +15,8 @@ const UserAgent = require('UserAgent');
 
 const isOSX = UserAgent.isPlatform('Mac OS X');
 
+const isSoftNewlineEvent = require('./isSoftNewlineEvent');
```

Comment:
Based on how imports are done in other files, I think this would be better right underneath `const UserAgent = require('UserAgent');` above, and using the custom `require` format with only the module name, 

```suggestion
const isSoftNewlineEvent = require('isSoftNewlineEvent');
```